content/id/docs/tasks/administer-cluster/reserve-compute-resources.md

```diff
@@ -0,0 +1,232 @@
+diatur untuk mereservasi jumlah ID proses untuk _daemon_ sistem kubernetes.
+
+Secara opsional, kamu dapat memaksa _daemon_ sistem melalui setelan `kube-reserved`.
+Ini dilakukan dengan menspesifikasikan _parent_ cgroup sebagai nilai dari _flag_ `--kube-reserved-cgroup` pada kubelet.
+
+Kami merekomendasikan _daemon_ sistem kubernetes untuk ditempatkan pada
+tingkatan cgroup yang tertinggi (contohnya, `runtime.slice` pada mesin systemd).
+Secara ideal, setiap _daemon_ sistem sebaiknya dijalankan pada _child_ cgroup
+di bawah _parent_ ini. Lihat [dokumentasi]((https://git.k8s.io/community/contributors/design-proposals/node/node-allocatable.md#recommended-cgroups-setup))
+untuk mengetahui rekomendasi hierarki cgroup secara detail.
+
+Catatan: kubelet **tidak membuat** `--kube-reserved-cgroup` jika cgroup
+yang diberikan tidak ada pada sistem. Jika cgroup yang tidak valid diberikan,
+maka kubelet akan mengalami kegagalan.
+
+### System Reserved
+
+- **_Flag_ Kubelet**: `--system-reserved=[cpu=100m][,][memory=100Mi][,][ephemeral-storage=1Gi][,][pid=1000]`
+- **_Flag_ Kubelet**: `--system-reserved-cgroup=`
+
+
+`system-reserved` berfungsi untuk mengetahui reservasi sumber daya untuk
+_daemon_ sistem pada OS, seperti `sshd`, `udev`, dan lainnya. `system-reserved` sebaiknya
+mereservasi memori untuk kernel juga, karena memori kernel tidak termasuk dalam
+hitungan kalkulasi Pod pada Kubernetes. Kami juga merekomendasikan reservasi sumber daya
+untuk sesi (_session_) login pengguna (contohnya, `user.slice` di dalam dunia systemd).
+
+### Melakukan Reservasi Daftar CPU secara Eksplisit
+{{< feature-state for_k8s_version="v1.17" state="stable" >}}
+
+- **_Flag_ Kubelet**: `--reserved-cpus=0-3`
+
+`reserved-cpus` berfungsi untuk mendefinisikan [cpuset](https://www.kernel.org/doc/Documentation/cgroup-v1/cpusets.txt) secara eksplisit untuk
+_daemon_ sistem OS dan _daemon_ sistem kubernetes. `reserved-cpus` dimaksudkan untuk
+sistem-sistem yang tidak mendefinisikan tingkatan cgroup tertinggi secara terpisah untuk
+_daemon_ sistem OS dan _daemon_ sistem kubernetes yang berkaitan dengan sumber daya cpuset.
+
+Jika Kubelet **tidak memiliki** `--system-reserved-cgroup` dan `--kube-reserved-cgroup`,
+cpuset akan diberikan secara eksplisit oleh `reserved-cpus`, yang akan menimpa definisi
+yang diberikan oleh opsi `--kube-reserved` dan `--system-reserved`.
+
+Opsi ini dirancang secara spesifik untuk kasus-kasus Telco/NFV, dimana _interrupt_ atau _timer_
+yang tidak terkontrol bisa mempengaruhi performa dari beban kerja. Kamu dapat menggunakan
+opsi untuk untuk mendefinisikan cpuset secara eksplisit untuk _daemon_ sistem/kubernetes dan
+_interrupt_/_timer_, sehingga CPU sisanya dalam sistem akan digunakan untuk beban kerja saja,
+dengan dampak yang sedikit terhadap _interrupt_/_timer_ yang tidak terkontrol. Untuk
+memindahkan _daemon_ sistem, _daemon_ Kubernetes serta _interrrupt_/_timer_ kubernetes supaya
```

Comment:
Is Kubelet use capital "K" or small "k"? make sure consistent!
It is a very clear example of "inconsistency" :)
```suggestion
memindahkan _daemon_ sistem, _daemon_ Kubernetes serta _interrrupt_/_timer_ Kubernetes supaya
```